lib/resque.rb

```diff
@@ -483,7 +483,7 @@ def remove_worker(worker_id)
   # Returns a hash, similar to redis-rb's #info, of interesting stats.
   def info
     return {
-      :pending   => queue_sizes.inject(0) { |sum, (queue_name, queue_size)| sum + queue_size },
+      :pending   => queue_sizes.inject(0) { |sum, (_queue_name, queue_size)| sum + queue_size },
```

Comment:
Isn't `_queue_name` an unused variable? If it's unused, usually the convention is to name it `_`, no?